www/js/survey/enketo/enketo-add-note-button.js

```diff
@@ -3,131 +3,121 @@
  */
 
 import angular from 'angular';
+import { getAdditionsForTimelineEntry, getUniqueEntries } from '../inputMatcher';
 
 angular
   .module('emission.survey.enketo.add-note-button', [
     'emission.services',
     'emission.survey.enketo.answer',
-    'emission.survey.inputmatcher',
   ])
-  .factory(
-    'EnketoNotesButtonService',
-    function (InputMatcher, EnketoSurveyAnswer, Logger, $timeout) {
-      var enbs = {};
-      console.log('Creating EnketoNotesButtonService');
-      enbs.SINGLE_KEY = 'NOTES';
-      enbs.MANUAL_KEYS = [];
+  .factory('EnketoNotesButtonService', function (EnketoSurveyAnswer, Logger, $timeout) {
+    var enbs = {};
+    console.log('Creating EnketoNotesButtonService');
+    enbs.SINGLE_KEY = 'NOTES';
+    enbs.MANUAL_KEYS = [];
 
-      /**
-       * Set the keys for trip and/or place additions whichever will be enabled,
-       * and sets the name of the surveys they will use.
-       */
-      enbs.initConfig = function (tripSurveyName, placeSurveyName) {
-        enbs.tripSurveyName = tripSurveyName;
-        if (tripSurveyName) {
-          enbs.MANUAL_KEYS.push('manual/trip_addition_input');
-        }
-        enbs.placeSurveyName = placeSurveyName;
-        if (placeSurveyName) {
-          enbs.MANUAL_KEYS.push('manual/place_addition_input');
-        }
-      };
+    /**
+     * Set the keys for trip and/or place additions whichever will be enabled,
+     * and sets the name of the surveys they will use.
+     */
+    enbs.initConfig = function (tripSurveyName, placeSurveyName) {
+      enbs.tripSurveyName = tripSurveyName;
+      if (tripSurveyName) {
+        enbs.MANUAL_KEYS.push('manual/trip_addition_input');
+      }
+      enbs.placeSurveyName = placeSurveyName;
+      if (placeSurveyName) {
+        enbs.MANUAL_KEYS.push('manual/place_addition_input');
+      }
+    };
```

Comment:
wait, why do we have all these extraneous whitespace changes? I thought that once we used prettier, we wouldn't need this any more.

@jiji14 do you need to re-pull the branch?